api/api-pipeline-operations-v1/src/main/java/com/thoughtworks/go/apiv1/pipelineoperations/PipelineOperationsControllerV1Delegate.java

```diff
@@ -100,9 +100,12 @@ public void setupRoutes() {
 
     public String pause(Request req, Response res) throws IOException {
         HttpLocalizedOperationResult result = new HttpLocalizedOperationResult();
-        JsonReader requestBody = GsonTransformer.getInstance().jsonReaderFrom(req.body());
+        String pauseCause = null;
         String pipelineName = req.params("pipeline_name");
-        String pauseCause = requestBody.optString("pause_cause").orElse(null);
+        if (!StringUtils.isBlank(req.body())) {
```

Comment:
I'd probably handle the null in the `GsonTransformer` instead of here — avoids similar problem in other cases.